x/oracle/keeper/owasm.go

```diff
@@ -93,7 +98,10 @@ func (k Keeper) PrepareRequest(ctx sdk.Context, r types.RequestSpec, ibcSource *
 	if len(req.RawRequests) == 0 {
 		return 0, types.ErrEmptyRawRequests
 	}
-
+	// Collect ds fee
+	if _, err := k.CollectFee(ctx, feePayer, r.GetFeeLimit(), askCount, req.RawRequests); err != nil {
+		return 0, err
```

Comment:
Will `collector.Collected()` returned by `CollectFee()` be used in the future?

Reply:
Probably but for now we use this for testing purposes.